jupyterhub_moss/batch_script.sh

```diff
@@ -11,6 +11,7 @@
 {% endif %}{% if nnodes     %}#SBATCH --nodes={{nnodes}}
 {% endif %}{% if ntasks     %}#SBATCH --ntasks-per-node={{ntasks}}
 {% endif %}{% if exclusive  %}#SBATCH --exclusive
+{% endif %}{% if output is not defined or not output %}#SBATCH --output=/dev/null
```

Comment:
`is not defined` is an interesting construct. Does it come from the template ?

Reply:
Yes it is a template stuff.

Reading it back, I have to check with I didn't simply put:
```suggestion
{% endif %}{% if not output %}#SBATCH --output=/dev/null
```

Reply:
Depends how `output` is managed if not set.

If it is set to `None` then you will indeed get the same behaviour (`not None` being `True`)

Reply:
Indeed ```python
{% if not output %}#SBATCH --output=/dev/null
``` is fine.